std_msgs/Header header_ros
std_msgs/UInt8 frame_header1
std_msgs/UInt8 frame_header2
# std_msgs/UInt8 indicator_of_frame
std_msgs/UInt8 frame_count
std_msgs/UInt8 frame_length
std_msgs/UInt8 flag_for_launch
std_msgs/UInt8 valid_flag_for_gps
std_msgs/UInt16 gps_cycle
std_msgs/UInt32 tic_toc
std_msgs/Float32 latitude
std_msgs/Float32 longitude
std_msgs/Float32 altitude
# std_msgs/Float32 velocity_to_north
# std_msgs/Float32 velocity_to_east
# std_msgs/Float32 velocity_to_sky
# std_msgs/Float32 armX
# std_msgs/Float32 armY
# std_msgs/Float32 armZ
# std_msgs/Float32 localization_accuracy_factor
std_msgs/UInt16 orientation_uav
std_msgs/UInt16 checksum

